pkg/querier/blocks_store_queryable.go

```diff
@@ -958,12 +958,7 @@ func (q *blocksStoreQuerier) receiveMessage(c BlocksStoreClient, stream storegat
 
 	// Response may either contain series, streaming series, warning or hints.
 	if s := resp.GetSeries(); s != nil {
-		// Take a safe copy of every label.
-		for i, l := range s.Labels {
-			s.Labels[i].Name = strings.Clone(l.Name)
-			s.Labels[i].Value = strings.Clone(l.Value)
-		}
-		mySeries = append(mySeries, s)
+		mySeries = append(mySeries, s.CloneUnsafe())
```

Comment:
This doesn't seem correct, `CloneUnsafe()` doesn't return anything, right?

Reply:
This `CloneUnsafe` method does return the `Series`. I can make the other `CloneUnsafe` implementations do the same, for consistency.

Reply:
In the latest revision they all return the method receiver.